dashboard/app/views/shared/_user_header.html.haml

```diff
@@ -33,6 +33,20 @@
       %span= t('nav.user.signin')
 
 :javascript
```

Comment:
Are we able to move this (and the rest of the JS in this file) into our JS pipeline?